src/devices/IS31FL3730/MatrixMode.cs

```diff
@@ -0,0 +1,26 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+namespace Iot.Device.IS31FL3730
+{
+    /// <summary>
+    /// Which of the two matrices should the controller drive.
+    /// </summary>
+    public enum MatrixMode
+    {
+        /// <summary>
+        /// Drive only Matrix 1.
+        /// </summary>
+        Matrix1Only,
+
+        /// <summary>
+        /// Drive only Matrix 2.
+        /// </summary>
+        Matrix2Only,

```

Comment:
and to give you the example of how to to this enum properly to be used directly later on in the code:

```csharp
Matrix2Only = 0b0000_1000,
```

so you can apply this pattern to all the enums you have.